src/python/build_management/build_manager.py

```diff
@@ -279,6 +276,9 @@ def _set_random_fuzz_target_for_fuzzing_if_needed(fuzz_targets, target_weights):
     logs.log_error('No fuzz targets found. Unable to pick random one.')
     return None
 
+  fuzz_targets = list(fuzz_targets)
```

Comment:
probably move this line before line:275 since it can be iterator. environment.set_value can be here as-is.